lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -286,7 +286,7 @@ def extract_student(student, features):
         if include_enrollment_mode or include_verification_status:
             enrollment_mode = CourseEnrollment.enrollment_mode_for_user(student, course_key)[0]
             if include_verification_status:
-                student_dict['verification_status'] = SoftwareSecurePhotoVerification.verification_status_for_user(
+                student_dict['verification_status'] = IDVerificationService.verification_status_for_user(
                     student,
                     course_key,
```

Comment:
Method sig changed and no longer takes course key. Is there a test for this? I would have expected it to fail

Reply:
course key wasn't being used in the function's implementation soI removed it.